File: 283-MoveZeroes/283-MoveZeroes.py
# Last updated: 5/27/2026, 12:35:07 PM
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        l=[]
        s=len(nums)
        for i in range(len(nums)):
            if nums[i]!=0:
                l.append(nums[i])
        n=len(l)
        for i in range(n):
            nums[i]=l[i]      
        for i in range(n,s):
            nums[i]=0
        return nums

File: 283-MoveZeroes/283-MoveZeroes.java
// Last updated: 5/27/2026, 12:35:00 PM
class Solution {
    public void moveZeroes(int[] nums) {
        int left=0,right=0;
        while(right<nums.length){
            if(nums[right]!=0){
                int t=nums[left];
                nums[left] = nums[right];
                nums[right]=t;
                left++;

            }
            right++;
        }
    }
}